lib/events/filelog.go

```diff
@@ -532,6 +541,31 @@ func (l *FileLog) findInFile(fn string, query url.Values, total *int, limit int)
 	return retval, nil
 }
 
+// FilterOutSessionEnd returns whether to filter out session end event
+// determined by whether the input slice contains session end event session ID
```

Comment:
I read `FilterOutSessionEnd()` as a verb at the call site, so I kind of expected it to be doing the actual filtering rather than being a predicate.

I'm not sure I have a better name though. All I've got is `ShouldFilterOutSessionEnd()`, which is terrible.

Reply:
I agree, I would not have modularized it, but doing it for the sake of keeping code DRY. I wonder if keeping the logic within each method is better in this case. I can't think of a better name either, so I will change to `ShouldFilterOutSessionEnd()` for now